bps/serial.go

```diff
@@ -52,3 +53,19 @@ func (b *BPS) Scan(value interface{}) error {
 func (b *BPS) Value() (driver.Value, error) {
 	return b.String(), nil
 }
+
+// UnmarshalText implements the encoding.TextUnmarshaler interface.
+func (b *BPS) UnmarshalText(text []byte) error {
```

Comment:
Please add the compiler's check about the implementation of encoding.TextUnmarshaler interface.

Reply:
@iwata  I am going to do so. 
By the way, I think we can also unite like following. Which do you prefer?

```go
var _ interface {
	sql.Scanner
	driver.Valuer
	encoding.TextUnmarshaler
} = (*BPS)(nil)
```

Reply:
As for me, I wouldn't mind either way 😉

Reply:
OK, I am going to unite it :pray: